src/app/views/jobs/jobs.component.ts

```diff
@@ -27,19 +36,41 @@ export class JobsComponent extends TranslationListener implements OnInit {
 
   ngOnInit() {
     this.loadData();
+
+    this.initLocation();
   }
 
   loadData() {
-    this.jobProxy.getJobs({include: 'owner,company,hourly_pay,company.company_images', 'filter[filled]': false, 'page[number]': this.page.toString()})
+    this.jobProxy.getJobs({ include: 'owner,company,hourly_pay,company.company_images', 'filter[filled]': false, 'page[number]': this.page.toString() })
       .then(result => {
         this.jobs = result.data;
         this.totalJobs = result.total;
       });
   }
 
+  initLocation() {
+    this.geolocationService.getLocation().subscribe(
+      position => {
+        this.mapUserLocation = new MapLocation({
+          longitude: position.coords.longitude,
+          latitude: position.coords.latitude
+        });
+        this.mapLocation = this.mapUserLocation;
+        this.mapZoom = 12;
+      },
+      error => {
+        console.log(error);
```

Comment:
True :)

I always err on the side of that giving too much feedback is better than not enough :) So in this case I think its great that to display:

```
Browser does not support location services
You have rejected access to your location
Unable to determine your location
Service timeout has been reached
```
The user might have misunderstood or miss clicked and then confirming what they've done can spare the user some frustration.

So my suggestion would be to just display it in a notice-bar/tooltip beneath the button or something, @bex1 you think that can be added and/or if its a good idea in your opinion? :)